controllers/authController.js

```diff
@@ -0,0 +1,227 @@
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const Joi = require("joi");
+const gravatar = require("gravatar");
+const User = require("../models/user");
+const jimp = require("jimp");
+const path = require("path");
+const fs = require("fs/promises");
+const nodemailer = require("nodemailer");
+const { v4: uuidv4 } = require("uuid");
+require("dotenv").config({ path: "./mongo.env" });
+
+const registerSchema = Joi.object({
+  email: Joi.string().email().required(),
+  password: Joi.string().min(6).required(),
+});
+
+const loginSchema = Joi.object({
+  email: Joi.string().email().required(),
+  password: Joi.string().required(),
+});
+
+const sendVerificationEmail = async (email, verificationToken) => {
+  try {
+    const transport = nodemailer.createTransport({
+      host: "smtp.meta.ua",
```

Comment:
why do you need it here? you have already defined nodemailer transport. in app.js. add  it in separate file and just invoke here